app/controllers/books_controller.rb

```diff
@@ -3,11 +3,11 @@ class BooksController < ApplicationController
 
   def index
     books = Book.all
-    render_paginated json: books
+    render_paginated json: books, each_seralizer: BookSerializer
   end
 
   def show
     book = Book.find(params[:id])
-    render_paginated json: book
+    render json: book, each_seralizer: BookSerializer
```

Comment:
`each_serializer`. Watch out for typos in params, if a parameter is in hash form it will almost surely go unchecked (unless it's forcefully required in the other side, for instance doing something like
`throw 'nope' unless params[:each_serializer].present?`
which is not that common)

In this case it'll work but use the default serializer instead, and those bugs suck.